charts/px-backup/templates/pxcentral-pod-list.yaml

```diff
@@ -0,0 +1,27 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: pods-list
+  namespace: {{ .Values.namespace }} 
+  labels:
+{{- include "px-backup.labels" . | nindent 4 }}
+rules:
+- apiGroups: ["*"]
+  resources: ["*"]
+  verbs: ["*"]
```

Comment:
cluster role says pods-list, but takes all permissions. Please fix.

Reply:
Changed and verified.